### PR TITLE
ENH: Added getters for parameters and cameras calibration

### PR DESCRIPTION
**Pull request by atracsys-sbt**
Added methods to access, from the Atracsys wrapper, the values of device parameters set in the config file
Added a function to access the cameras calibration (intrinsic parameters from the cameras)

### CODE REVIEW

File: src/PlusDataCollection/Atracsys/vtkPlusAtracsysTracker.h [hunk 1]
```diff
@@ -79,6 +84,9 @@ class vtkPlusDataCollectionExport vtkPlusAtracsysTracker : public vtkPlusDevice
   vtkPlusAtracsysTracker();
   ~vtkPlusAtracsysTracker();
 
+  // helper to translate option names from Plus nomenclature to Atracsys' one
+  bool translateOptionName(const std::string& optionName, std::string& translatedOptionName);
```
Comment (Sunderlandkyl) [inline src/PlusDataCollection/Atracsys/vtkPlusAtracsysTracker.h:88]: Method name should start with a capital letter.

File: src/PlusDataCollection/Atracsys/vtkPlusAtracsysTracker.h [hunk 2]
```diff
@@ -32,6 +32,9 @@ class vtkPlusDataCollectionExport vtkPlusAtracsysTracker : public vtkPlusDevice
   /* Get device type */
   std::string GetDeviceType();
 
+  /* Get cameras calibration */
+  PlusStatus GetCamerasCalibration(std::vector<float>& calib);
```
Comment (Sunderlandkyl) [inline src/PlusDataCollection/Atracsys/vtkPlusAtracsysTracker.h:36]: Add a comment describing the content of calib.

File: src/PlusDataCollection/Atracsys/AtracsysTracker.h [hunk 3]
```diff
@@ -150,6 +152,9 @@ class AtracsysTracker
   /*! */
   ATRACSYS_RESULT GetDeviceId(uint64_t& id);
 
+  /*! */
+  ATRACSYS_RESULT GetCamerasCalibration(std::vector<float>& calib);
```
Comment (Sunderlandkyl) [inline src/PlusDataCollection/Atracsys/AtracsysTracker.h:156]: Add a comment describing the method/content of calib.

**Review by Sunderlandkyl — COMMENTED** [review]
Thanks, just added a couple of comments.

**Comment by atracsys-sbt** [comment]
While commenting I realized how bad was the getter argument, so I improved it a bit.

**Comment by Sunderlandkyl** [comment]
Thanks!

**Pull request merged**